docs/fsharp/using-fsharp-on-azure/package-management.md

```diff
@@ -14,16 +14,16 @@ Obtaining packages for Azure development is easy when you use a package manager.
 If you're using [Paket](https://fsprojects.github.io/Paket/) as your dependency manager, you can use the `paket.exe` tool to add Azure dependencies. For example:
 
 ```console
-> paket add nuget WindowsAzure.Storage
+> paket add nuget Azure.Storage.Blobs
```

Comment:
To normalize for cross-plat development, we should focus on using `Packet` as a `dotnet` tool.    see:  [https://fsprojects.github.io/Paket/installation.html](https://fsprojects.github.io/Paket/installation.html)